app/questionnaire/router.py

```diff
@@ -228,7 +249,7 @@ def _get_first_incomplete_location_in_section(self, routing_path):
                     list_name=routing_path.list_name,
                 )
 
-    def _get_allowable_path(self, routing_path):
+    def _get_allowable_path(self, routing_path: RoutingPath) -> List:
```

Comment:
We should use `list` and we should return the types.

Reply:
Added return type and `list` thanks!